docs/features/autoupdater.rst

```diff
@@ -16,10 +16,16 @@ be set on the command line, or it can be taken from the ``site.mk``.
 
 The priority defines the maximum number of days that may pass between releasing an update and installation
 of the images. The update probability will start at 0 after the release time mentioned in the manifest
+
+>  where and how is this mentioned?
```

Comment:
I know, but strangely, by the time I created this PR, there were no issues link any more, so I created a PR.

I will amend the correct enhancement here later, so it will become a good PR ;)